lib/type/any.ex

```diff
@@ -8,13 +8,31 @@ defmodule Litmus.Type.Any do
   @type t :: %__MODULE__{
           required: boolean
         }
+
+  @spec add_required_errors(map, binary, t) :: {:ok, map} | {:error, String.t()}
+  def add_required_errors(params, field, %Litmus.Type.Any{required: true}) do
```

Comment:
You should use `%__MODULE__{required: true}`. `__MODULE__` will always be the name of the current module. If you ever rename the module, you won't have to edit a bunch of code. And it's more idiomatic.